src/js/msg.js

```diff
@@ -15,6 +15,31 @@ export const wrapError = error => ({
   }, error), // passing custom properties e.g. `error.index`
 });
 
+// SECURITY: Restrict messages to trusted content scripts
```

Comment:
This is unnecessary because onMessage doesn't receive messages from other extensions, it's the job for onMessageExternal, which we don't use. If, due to a bug in the browser, someone manages to break through world isolation barrier they'll be able to spoof all those props anyway.